testcases/TC14.1-all-dna-1.md

```diff
@@ -0,0 +1,53 @@
+---
+# The layout must be 'testcase'; DO NOT Change
+layout: testcase
+
+# Brief, descriptive title for the test case
+title: Link Purpose(In Context) - no links (TC14-1-all-dna-1)
+
+# The Test Case ID should follow the pattern: 
+# TC[Baseline Test Procedure #]-[Test Instruction #]-
+# [Expected Result (pass/fail/dna)]-[example #], e.g., TC05.1-1-fail-1
+tcid: TC14.1-all-dna-1
+
+# Description of the Test Case, the included code sample, test considerations,
+# and rationale for the expected result according to the applicable ICT
+# Baseline test
+descr: |
+  No link is detected.
+
+
+# Reference and link to the applicable ICT Baseline test
+app-basline: |
+ [14.1 Test Procedure for Link Purpose (In Context)](https://section508coordinators.github.io/ICTTestingBaseline/14Links.html#141-test-procedure-for-link-purpose-in-context)
+
+ **Baseline Test ID:** 14.1-LinkPurpose
+ 
+ **Test Instruction:** 1
+
+# Expected result that the ICT Baseline would predict
+# [Pass | Fail | DNA]
+result: DNA
+
+# Brief description of the rationale for the expected result
+result-descr: The code sample shows that there are no links present.
+
+# URL for the code sample
+# In the sample code file, add id="tc_code" to the 
+# element that contains the relevant code snippet.
+#
+# Then upload the code sample to the 'testfiles' folder 
+# and provide the link (and only the url) below.
+sample: /testfiles/TF14/14.1-all-dna-1.html
+
+# Table of test instructions, including the following table headers: 
+# Test Instruction #; Instruction Detail; Expected Test Case Result
+#
+# Include the table in the content section below
+---
+| Test Instruction | Instruction Detail | Expected Test Case Result |
+|------------------|--------------------|---------------------------|
+| IC-1|All links including those that are scripted elements and assigned a role of a link.|
```

Comment:
explain none found in the 3rd column:
| | | No links found. DNA |

Reply:
Please add the third column: No links found.

Reply:
The 3rd column has been updated to show "No links found." please see below and the updated test file.

|IC-1 | All links including those that are scripted elements and assigned a role of a link. | No links found.|